integration/spark/spark33/src/main/java/io/openlineage/spark33/agent/lifecycle/plan/CreateReplaceDatasetBuilder.java

```diff
@@ -168,9 +168,10 @@ private List<OpenLineage.OutputDataset> apply(
           .ifPresent(
               version -> DatasetVersionUtils.buildVersionOutputFacets(context, builder, version));
     }
-
     CatalogUtils3.getStorageDatasetFacet(context, catalog, tableProperties)
         .map(storageDatasetFacet -> builder.getFacets().storage(storageDatasetFacet));
+    CatalogUtils3.getCatalogDatasetFacet(context, catalog, tableProperties)
```

Comment:
How about:
 * `io.openlineage.spark33.agent.lifecycle.plan.CreateReplaceDatasetBuilder`
 * `io.openlineage.spark34.agent.lifecycle.plan.column.CreateReplaceInputDatasetBuilder`
 * `io.openlineage.spark35.agent.lifecycle.plan.CreateReplaceOutputDatasetBuilder`
 ?

Would it make sense to merge: `io.openlineage.spark3.agent.lifecycle.plan.catalog.CatalogUtils3#getStorageDatasetFacet` and `io.openlineage.spark3.agent.lifecycle.plan.catalog.CatalogUtils3#getCatalogDatasetFacet` into `loadCatalogFacets` or alike?

Reply:
fixed, except CreateReplaceInputDatasetBuilder since it's related to cll